sympy/core/local_random.py

```diff
@@ -0,0 +1,32 @@
+"""
+This module is a copy of python's random module but using
+a Random instance for all supported methods. It is needed
+for replicating results when using seeds.
+"""
+
+import random as std_random
+
+Random = std_random.Random
+SystemRandom = std_random.SystemRandom
+_inst = Random()
+seed = _inst.seed
+random = _inst.random
+uniform = _inst.uniform
+triangular = _inst.triangular
+randint = _inst.randint
+choice = _inst.choice
+randrange = _inst.randrange
+sample = _inst.sample
+shuffle = _inst.shuffle
+normalvariate = _inst.normalvariate
+lognormvariate = _inst.lognormvariate
+expovariate = _inst.expovariate
+vonmisesvariate = _inst.vonmisesvariate
+gammavariate = _inst.gammavariate
+gauss = _inst.gauss
+betavariate = _inst.betavariate
+paretovariate = _inst.paretovariate
+weibullvariate = _inst.weibullvariate
```

Comment:
Do we need all these functions?

Reply:
I'm not objecting to them, just asking. Are these functions all needed or is this just the full list in the random module?

Reply:
I think it's better just to have the functions that are needed here.

Reply:
I agree. I would also add these to the compatibility file rather than a new one.

Reply:
I know very well python except for modules and submodules, thus I am not sure how to plug `local_random` inside `compatibility`. Take into account that we need two types of imports:

1. `import whatever_we_agree as random`
2. `from whatever_we_agree import random, shuffle`

Is it correct if I do it with a class as below? Otherwise, can you provide me a small example on how to support the two types of imports while having the random module inside compatibility?

 ```python
# inside compatibility
...
class random:
    random = ...
    shuffle = ...
...
```